src/tiledimage.js

```diff
@@ -844,7 +844,7 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
             return;
         }
         if (immediately) {
-            this._degreesSpring.resetTo(degrees);
+          this._degreesSpring.resetTo(degrees);
```

Comment:
Looks like you accidentally removed a couple of spaces here.

Reply:
Fixed.